app/views/about_users/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+%p
+  = f.label :name, 'Nama:'
+%p
+  = f.text_field :name
```

Comment:
Actually, why not just take the name from the user's full name field?

Reply:
Take it as the default value or make it permanent?

Reply:
Hm I think make it permanent. i.e. just don't have the name entry in this model. Just read it from the user model instead.

Reply:
@donjar So, we assume that the full name is always correct (at least for panitia/admin)?

Reply:
Yep let's do that.